packages/frontend/src/config/environmentDefaults/development.js

```diff
@@ -53,6 +53,7 @@ export default {
     FARMING_CLAIM_GAS: parseNearAmount('0.00000000015'),
     FARMING_CLAIM_YOCTO: '1',
     REF_FINANCE_API_ENDPOINT: 'https://testnet-indexer.ref-finance.com/',
+    TOKEN_BLACKLIST_ENDPOINT: '',
```

Comment:
Because it is for mainnet. Won't it throws some kind of RPC error? In my mind we can't join this kind of info for different networks.
But I suppose we can just remove empty vars.